### ISSUE TITLE
[BUG] Several npm audit vulnerabilities issues

### ISSUE THREAD

**Issue by Divlo** [soketi/soketi#807]
There are several vulnerabilities detected by `npm audit`:

```sh
$ npm audit
# npm audit report

decode-uri-component  <0.2.1
decode-uri-component vulnerable to Denial of Service (DoS) - https://github.com/advisories/GHSA-w573-4hg7-7wgq
fix available via `npm audit fix`
node_modules/decode-uri-component

json5  2.0.0 - 2.2.1
Severity: high
Prototype Pollution in JSON5 via Parse Method - https://github.com/advisories/GHSA-9c47-m6qq-7p4h
fix available via `npm audit fix`
node_modules/json5

knex  <2.4.0
Severity: high
Knex.js has a limited SQL injection vulnerability - https://github.com/advisories/GHSA-4jv9-3563-23j3
fix available via `npm audit fix --force`
Will install knex@2.4.2, which is a breaking change
node_modules/knex

luxon  3.0.0 - 3.2.0
Severity: high
Luxon Inefficient Regular Expression Complexity vulnerability - https://github.com/advisories/GHSA-3xq5-wjfh-ppjc
fix available via `npm audit fix`
node_modules/luxon

4 vulnerabilities (1 low, 3 high)

To address issues that do not require attention, run:
  npm audit fix

To address all issues (including breaking changes), run:
  npm audit fix --force
```

**Comment by parallels999** [comment]
already without support

**Comment by Itemshopp** [comment]
I should probably first add a message here before creating a new issue : 

Here are for now the new `npm audit report`

```
# npm audit report

knex  <2.4.0
Severity: high
Knex.js has a limited SQL injection vulnerability - https://github.com/advisories/GHSA-4jv9-3563-23j3
No fix available
node_modules/knex
  @soketi/soketi  *
  Depends on vulnerable versions of knex
  Depends on vulnerable versions of pm2
  node_modules/@soketi/soketi

semver  <7.5.2
Severity: moderate
semver vulnerable to Regular Expression Denial of Service - https://github.com/advisories/GHSA-c2qf-rxjj-qqgw
No fix available
node_modules/@opencensus/core/node_modules/semver
node_modules/@opencensus/propagation-b3/node_modules/semver
node_modules/@pm2/agent/node_modules/semver
node_modules/@pm2/io/node_modules/semver
node_modules/async-listener/node_modules/semver
  @opencensus/core  <=0.0.21
  Depends on vulnerable versions of semver
  node_modules/@opencensus/core
  node_modules/@opencensus/propagation-b3/node_modules/@opencensus/core
    @opencensus/propagation-b3  0.0.3 - 0.0.21
    Depends on vulnerable versions of @opencensus/core
    node_modules/@opencensus/propagation-b3
      @pm2/io  *
      Depends on vulnerable versions of @opencensus/core
      Depends on vulnerable versions of @opencensus/propagation-b3
      Depends on vulnerable versions of semver
      node_modules/@pm2/io
        pm2  >=3.0.0
        Depends on vulnerable versions of @pm2/agent
        Depends on vulnerable versions of @pm2/io
        node_modules/pm2
  @pm2/agent  >=0.5.3
  Depends on vulnerable versions of semver
  node_modules/@pm2/agent
  async-listener  >=0.6.4
  Depends on vulnerable versions of semver
  node_modules/async-listener

word-wrap  *
Severity: moderate
word-wrap vulnerable to Regular Expression Denial of Service - https://github.com/advisories/GHSA-j8xg-fqg3-53r7
fix available via `npm audit fix`
node_modules/word-wrap
  optionator  0.8.3 - 0.9.1
  Depends on vulnerable versions of word-wrap
  node_modules/escodegen/node_modules/optionator

11 vulnerabilities (9 moderate, 2 high)

To address issues that do not require attention, run:
  npm audit fix

Some issues need review, and may require choosing
a different dependency.
```

A little bit more vulnerabilities than 3 month ago

**Comment by rennokki** [comment]
Most of them were addressed in 1.6.0
